frontend/store/auctions.ts

```diff
@@ -42,6 +46,7 @@ export const state = (): State => ({
     error: null,
     restartingAuctionsIds: [],
     lastUpdated: undefined,
+    selectedAuctionId: undefined,
```

Comment:
We shouldn't store this value, since it can be derived from the router

Reply:
This was left over! My bad 😄